docs/07_admin/02_RESTadmin.rst

```diff
@@ -0,0 +1,897 @@
++-----------+-----------+-----------------------------------------------+---------------------------+
+
+
+**Response Payload**
+
+The response payload is the newly created audit trail entry.
+
++-----------+-----------+-----------------------------------------------+---------------------------+
+| Name      | Type      | Description                                   | Example                   |
++===========+===========+===============================================+===========================+
+| timestamp | timestamp | The timestamp when the audit trail |br|       | 2018-04-16 14:33:18.215   |
+|           |           | item was written.                             |                           |
++-----------+-----------+-----------------------------------------------+---------------------------+
+| source    | string    | The source of the audit trail entry.          | LocalMonitor              |
++-----------+-----------+-----------------------------------------------+---------------------------+
+| severity  | string    | The severity of the event that triggered |br| | FAILURE                   |
+|           |           | the audit trail entry to be written. |br|     |                           |
+|           |           | This will be one of SUCCESS, FAILURE, |br|    |                           |
+|           |           | WARNING or INFORMATION.                       |                           |
++-----------+-----------+-----------------------------------------------+---------------------------+
+| details   | object    | A JSON object that describes the detail |br|  | { "message" : |br|        |
+|           |           | of the audit trail event.                     | "Internal System Error" } |
++-----------+-----------+-----------------------------------------------+---------------------------+
+
+
+**Example**
+
+.. code-block:: console
+
+  $ curl -X POST http://localhost:8081/foglamp/audit \
+  -d '{ "severity": "FAILURE", "details": { "message": "Internal System Error" }, "source": "LocalMonitor" }'
+  { "source": "LocalMonitor",
+    "timestamp": "2018-04-17 11:49:55.480",
+    "severity": "FAILURE",
+    "details": { "message": "Internal System Error" }
+  }
+  $
+  $ curl -X GET http://localhost:8081/foglamp/audit?severity=FAILURE
+  { "totalCount": 1,
+    "audit": [ { "timestamp": "2018-04-16 18:32:28.427",
+                 "source"   :    "LocalMonitor",
+                 "details"  : { "message": "Internal System Error" },
+                 "severity" : "FAILURE" }
+             ]
+  }
+  $
+
+|br|
+
+
+Configuration Management
+========================
+
+Configuration management in an important aspect of the REST API, however due to the discoverable form of the configuration of FogLAMP the API itself is fairly small. Information of the configuration management and in particularly the types of the configuration parameters that can be supported see The FogLAMP Configuration Component.
```

Comment:
"Information of the configuration management"  does not make sense as a sentence.

I suspect what you are trying to say here Is that the configuration management interface is generic in nature and the precise entries that may be manipulated is dependant on the plugins and components installed and in use within a FogLAMP deployment.